src/main/TokenIO.js

```diff
@@ -212,6 +214,23 @@ export class TokenIO {
         return this.createMember(alias, CryptoEngine, 'BUSINESS');
     }
 
+    /**
+     * Creates a claimed member with a alias and a keypair, using the provided engine
+     *
+     * @param  {Object} alias - alias to set for member,
+     *                  falsy value or empty object for a temporary member without an alias
```

Comment:
will an alias ever be necessary in createClaimedMember?

Reply:
In the current appless case, no, but maybe in the future?